machineLearning/stockRecommendationSystem/stockRecommender/events/event.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This isn't so much a change but I think this will make it a lot easier when testing a few different inputs for future work. 

When you call ```sam local invoke``` it defaults to this ```events.json``` file as the input which just has a bunch of dummy data in it now because its the default one. This can be used though to mock your API calls for different scenarios. 

In your Lambda function you can access this data through the ```event``` object passed as an input to your handler. This will contain the information in this file which is data which could be passed through an API call. 

You can create more than just one and use ```sam local invoke --event <PATH_TO_FILE>``` to change the event. So you might have a file which will pass a userId who has environment stocks called ```environment_event.json``` another maybe that has lots of social stocks in ```social_event.json``` and so on. Will make it handier to test and will replicate the deployment a bit more as it will be an API call in the end this is run on. 

Have a read [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-invoke.html)  and [here](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-concepts.html#gettingstarted-concepts-event) for bit more info on events.

Reply:
That's really good information @WarrenK-design, thanks a million!